typescript/src/redemption.ts

```diff
@@ -0,0 +1,291 @@
+// @ts-ignore
+import bcoin from "bcoin"
+// @ts-ignore
+import hash160 from "bcrypto/lib/hash160"
+import { BigNumber } from "ethers"
+import {
+  createKeyRing,
+  RawTransaction,
+  UnspentTransactionOutput,
+  Client as BitcoinClient,
+} from "./bitcoin"
+import { Bridge } from "./bridge"
+
+/**
+ * Represents a redemption request.
+ */
+export interface RedemptionRequest {
+  /**
+   * The address of the redeemer.
+   */
+  redeemerAddress: string
```

Comment:
Step 1:
I think we should remove that field and introduce two other ones instead:
```
redeemer: Identifier
redeemerOutputScript: string
```
The `redeemerOutputScript` should NOT be prefixed with `0x`. Let's leave prefixing to the `Bridge` implementation.

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/199/commits/4b3be0a5f176ab3e5733f1876ff60fa6d3ca53a2.